client/src/app/AudioButton/AudioButton.js

```diff
@@ -1,5 +1,7 @@
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
+import { motion } from 'framer-motion'
+import { clickAnimation } from '../../services/animationVariants'
 import { PlayIcon, PauseIcon } from '../Icons/Icons'
 
 AudioButton.propTypes = {
```

Comment:
Could the ``onClick`` prop be reqiured or do you have an use case, where the button does not need a click function passed to it?